contracts/SafeProtocolMediator.sol

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.18;
+import {ISafeProtocolMediator} from "./interfaces/Mediator.sol";
+import {ISafeProtocolModule} from "./interfaces/Components.sol";
+
+import {ISafe} from "./interfaces/Accounts.sol";
+import {SafeProtocolAction, SafeTransaction, SafeRootAccess} from "./DataTypes.sol";
+import {Ownable2Step} from "@openzeppelin/contracts/access/Ownable2Step.sol";
+
+/**
+ * @title TODO
+ * @notice TODO
+ */
+contract SafeProtocolMediator is ISafeProtocolMediator, Ownable2Step {
```

Comment:
Why is this Ownable?

Reply:
This PR also considers tasks from https://github.com/5afe/safe-protocol/issues/6#issue-1760196945
But, it doesn't completely address tasks in #6 to avoid fixing two issues in one PR.

Reply:
Yeah, I would remove it if we don't use it in this PR :)